src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__kfwd_contact_notes.sql

```diff
@@ -21,5 +21,8 @@ select
     cn.next_steps,
     cn.status,
     cn.type,
+
+    {{ date_to_fiscal_year(date_field="cn.date", start_month=7, year_source="start") }}
+    as academic_year,
```

Comment:
move this calc to `stg_kippadb__contact_note`

Reply:
turns out it was already there - fixed ✅